ci/bump_version.sh

```diff
@@ -1,12 +1,32 @@
 #!/bin/bash
 
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+YELLOW='\033[0;33m'
+BLUE='\033[0;34m'
+MAGENTA='\033[0;35m'
+CYAN='\033[0;36m'
+GRAY='\033[0;37m'
+
+# Reset
```

Comment:
I'd prefer you to change the comment to be more comprehensive, e.g. `Reset output color to the default`.

Reply:
Done 👍